knip.production.jsonc

```diff
@@ -1,9 +0,0 @@
-{
-	"$schema": "https://unpkg.com/knip@latest/schema.json",
-	"entry": ["src/**/index.ts!"],
-	"ignore": [
-		// https://github.com/JoshuaKGoldberg/ts-api-utils/issues/343
-		"src/usage/scopes.ts"
```

Comment:
Looks like this issue isn't resolved, we'll need to keep the ignore work around for now.

Reply:
Sorry, I forgot to actually upgrade knip itself, so that `"ignoreMembers": ["finish"]` becomes effective. Pushed this.